### ISSUE TITLE
Trouble with adding the project into oculus quest 

### ISSUE THREAD

**Issue by Joshitha30** [CS4240-Group6/ProjectNurture#12]
Hi
We are trying to put your project in oculus quest 2. Previously it worked,but now after updating the oculus quest it doesn't seem to work. It is showing errors and we are only able to view the scene but can't play it. So could you please help us figure out these bugs. I will add a screenshot of the bugs.
![IMG-20240507-WA0008](https://github.com/CS4240-Group6/ProjectNurture/assets/112505327/b2edec6e-2f15-4f04-8075-cdd014c500d9)

**Comment by case141** [comment]
Wow it has been a long time since I touched this project haha I'm surprised anyone even noticed it, not sure what the error is about but we did originally develop in on Oculus Quest so I'm not sure about compatibility with Oculus Quest 2, you updated your Oculus Quest device?

**Comment by Joshitha30** [comment]
Yeah , we were able to successfully have the project run in oculus quest 2
also before updating it. But after the update we are getting such errors.
And also your readme file says 'Navigate to where you have downloaded our
unity package and select it'
Could you please tell me what unity package are you talking about or where
I could find it?
